src/test/java/org/broadinstitute/barclay/help/DocWorkUnitTest.java

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.barclay.help;
+
+import org.broadinstitute.barclay.help.testinputs.TestArgumentContainer;
+import org.broadinstitute.barclay.help.testinputs.TestExtraDocs;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public class DocWorkUnitTest {
+
+    @Test
+    public void testPropertyMap() {
+        final DocWorkUnit workUnit = DocGenMocks
+                .createDocWorkUnit(getDefaultDocWorkUnitHandler(), TestExtraDocs.class);
+        Assert.assertTrue(workUnit.getRootMap().isEmpty());
+        Assert.assertEquals(workUnit.getProperty("test_property"), null);
+        workUnit.setProperty("test_property", "test_value");
+        Assert.assertEquals(workUnit.getProperty("test_property"), "test_value");
+        Assert.assertEquals(workUnit.getRootMap().size(), 1);
+    }
```

Comment:
The "test_property" and "test_value" strings should be factored out into constants.

Reply:
Done.